sites/fast-website/src/public/index.html

```diff
@@ -25,14 +25,24 @@
         <!-- hero-section:start -->
         <section class="section">
           <div class="section-content">
-            Hero section header
+            <fast-section-header>
+              <h5>ADAPTIVE UI SYSTEM, UTILITIES, & TOOLS</h5>
+              <h1 slot="badge">The adaptive interface system for modern web experiences</h1>
+              <p slot="body">Interfaces bult with FAST adapt to your design system and can be used with any modern UI Framework by leveraging industry standard Web Components.</p>
+              <fast-button slot="action" appearance="outline">Get start</fast-button>
+            </fast-section-header>
           </div>
         </section>
         <!-- hero-section:end -->
 
         <section class="section">
           <div class="section-content">
-            Page section header
+            <fast-section-header>
+              <h5>FLEXIBLE AND ADAPTABLE</h5>
+              <h1 slot="badge">One component, many faces</h1>
```

Comment:
Can you pass the badge component here - or change this from being an h1?

Reply:
updated